<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="InternalLoggerException.xml" source-language="en-US" target-language="ja-JP">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-efd8310" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">ae25fbb5-14a8-44f2-90c2-3c6d6fcaae7ff8b2b9ccd15696f94c20f8803131257b9bc9be43.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">f8b2b9ccd15696f94c20f8803131257b9bc9be43</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1b899505ed6d42225f2b3c9150bfdc080d0d1f9b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/28/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7.2,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:Microsoft.Build.BuildEngine.InternalLoggerException">
          <source>This exception is used to wrap an unhandled exception from a logger.</source>
          <target state="translated">この例外は、ロガーからのハンドルされない例外をラップするために使用されます。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:Microsoft.Build.BuildEngine.InternalLoggerException">
          <source>This exception can only be thrown by the <ph id="ph1">[!INCLUDE[vstecmsbuild](~/includes/vstecmsbuild-md.md)]</ph> engine.</source>
          <target state="translated">この例外は、によってのみスローされることができます、<ph id="ph1">[!INCLUDE[vstecmsbuild](~/includes/vstecmsbuild-md.md)]</ph>エンジンです。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:Microsoft.Build.BuildEngine.InternalLoggerException">
          <source>This exception stops the build.</source>
          <target state="translated">この例外は、ビルドを中止します。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" uid="T:Microsoft.Build.BuildEngine.InternalLoggerException">
          <source>Initializes the <ph id="ph1">&lt;see cref="T:Microsoft.Build.BuildEngine.InternalLoggerException" /&gt;</ph> class.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:Microsoft.Build.BuildEngine.InternalLoggerException" /&gt;</ph> クラスを初期化します。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="M:Microsoft.Build.BuildEngine.InternalLoggerException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:Microsoft.Build.BuildEngine.InternalLoggerException" /&gt;</ph> class.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:Microsoft.Build.BuildEngine.InternalLoggerException" /&gt;</ph> クラスの新しいインスタンスを初期化します。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="M:Microsoft.Build.BuildEngine.InternalLoggerException.#ctor(System.String)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">例外の原因を説明するエラー メッセージ。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:Microsoft.Build.BuildEngine.InternalLoggerException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:Microsoft.Build.BuildEngine.InternalLoggerException" /&gt;</ph> class using the specified values.</source>
          <target state="translated">指定した値を使用して、<ph id="ph1">&lt;see cref="T:Microsoft.Build.BuildEngine.InternalLoggerException" /&gt;</ph> クラスの新しいインスタンスを初期化します。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:Microsoft.Build.BuildEngine.InternalLoggerException.#ctor(System.String,System.Exception)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">例外の原因を説明するエラー メッセージ。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:Microsoft.Build.BuildEngine.InternalLoggerException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current excpetion.</source>
          <target state="translated">現在の例外の原因である例外。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:Microsoft.Build.BuildEngine.InternalLoggerException.#ctor(System.String,System.Exception)">
          <source>If the <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> parameter is not a null reference (<ph id="ph1">&lt;see langword="Nothing" /&gt;</ph> in Visual Basic), the current exception is raised in a <ph id="ph2">&lt;see langword="catch" /&gt;</ph> block that handles the inner exception.</source>
          <target state="translated"><bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> パラメーターが null 参照 (Visual Basic の場合は <ph id="ph1">&lt;see langword="Nothing" /&gt;</ph>) でない場合は、内部例外を処理する <ph id="ph2">&lt;see langword="catch" /&gt;</ph> ブロックで現在の例外が発生します。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:Microsoft.Build.BuildEngine.InternalLoggerException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:Microsoft.Build.BuildEngine.InternalLoggerException" /&gt;</ph> class using the specified values.</source>
          <target state="translated">指定した値を使用して、<ph id="ph1">&lt;see cref="T:Microsoft.Build.BuildEngine.InternalLoggerException" /&gt;</ph> クラスの新しいインスタンスを初期化します。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="P:Microsoft.Build.BuildEngine.InternalLoggerException.BuildEventArgs">
          <source>Gets the details of the build event, if any, that was being logged.</source>
          <target state="translated">ログに記録されていた構築イベントが存在する場合はその詳細を取得します。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="P:Microsoft.Build.BuildEngine.InternalLoggerException.BuildEventArgs">
          <source>The details of the build event that was being logged.</source>
          <target state="translated">ログに記録されていた構築イベントの詳細。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="P:Microsoft.Build.BuildEngine.InternalLoggerException.ErrorCode">
          <source>Gets the error code associated with this exception's message.</source>
          <target state="translated">この例外のメッセージに関連付けられたエラー コードを取得します。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT" uid="P:Microsoft.Build.BuildEngine.InternalLoggerException.ErrorCode">
          <source>The text for the error code.</source>
          <target state="translated">エラー コードのテキスト。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:Microsoft.Build.BuildEngine.InternalLoggerException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">スローされている例外に関するシリアル化済みオブジェクト データを保持している <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:Microsoft.Build.BuildEngine.InternalLoggerException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> that contains contextual information about the source or destination.</source>
          <target state="translated">転送元または転送先についてのコンテキスト情報を含む <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> です。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" uid="M:Microsoft.Build.BuildEngine.InternalLoggerException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Sets the <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> with information about the exception.</source>
          <target state="translated">例外に関する情報を含む <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> を設定します。</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="P:Microsoft.Build.BuildEngine.InternalLoggerException.HelpKeyword">
          <source>Gets the F1-help keyword associated with this error.</source>
          <target state="translated">このエラーに関連付けられている F1 ヘルプのキーワードを取得します。</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="P:Microsoft.Build.BuildEngine.InternalLoggerException.HelpKeyword">
          <source>The F1-help keyword associated with this error.</source>
          <target state="translated">このエラーに関連付けられている F1 ヘルプ キーワード。</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" uid="P:Microsoft.Build.BuildEngine.InternalLoggerException.InitializationException">
          <source>Gets whether an exception occurred during logger initialization.</source>
          <target state="translated">logger の初期化中に例外が発生したかどうかを取得します。</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="P:Microsoft.Build.BuildEngine.InternalLoggerException.InitializationException">
          <source><ph id="ph1">&lt;see langword="true" /&gt;</ph> if the exception occured during logger initialization, <ph id="ph2">&lt;see langword="false" /&gt;</ph> otherwise.</source>
          <target state="translated">logger の初期化中に例外が発生した場合は <ph id="ph1">&lt;see langword="true" /&gt;</ph>。それ以外の場合は <ph id="ph2">&lt;see langword="false" /&gt;</ph>。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>